pylintrc_tweaks

```diff
@@ -34,6 +34,53 @@ disable+ =
     no-name-in-module,
     unnecessary-lambda-assignment,
 
+    anomalous-backslash-in-string,
```

Comment:
We should add a comment before this section, something like "# Checked via ruff instead".